cms/static/cms/js/main.js

```diff
@@ -64,7 +64,8 @@
                     contentType: 'application/json; charset=utf-8',
                     dataType: 'json',
                     data: JSON.stringify(data),
-                    success: callback
+                    success: callback,
+                    global: data ? data.global: true    // Trigger global AJAX error handler or not
```

Comment:
*Note to reviewers*:  I have added this to support `global:false` in postJSON method.